sentry/src/main/java/io/sentry/NoOpSerializer.java

```diff
@@ -28,15 +28,17 @@ public Session deserializeSession(Reader reader) {
   }
 
   @Override
-  public SentryEnvelope deserializeEnvelope(InputStream inputStream) {
+  public Transaction deserializeTransaction(Reader reader) {
     return null;
   }
 
   @Override
-  public void serialize(SentryEvent event, Writer writer) {}
+  public SentryEnvelope deserializeEnvelope(InputStream inputStream) {
```

Comment:
Maybe this is enough:

```suggestion
  public @Nullable SentryEnvelope deserializeEnvelope(InputStream inputStream) {
```